.github/workflows/build-images-base.yaml

```
@@ -35,6 +35,14 @@ jobs:
       fail-fast: false
 
     steps:
+      - name: Checkout master branch to access local actions
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+        with:
+          ref: ${{ github.event.repository.default_branch }}
```

(nit) I think this could be simplified to 
```suggestion
          ref: ${{ github.repository.default_branch }}
```
but only 90% sure. (https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads)

---

Actually nah, I think I'm wrong.

---

Confirming I was wrong: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context
(was looking for this the other day but did not find it)